zest/releaser/pypi.py

```diff
@@ -407,6 +408,33 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string format for the tag name used in the release.
+
+        Override the default ``%(version)s`` in ~/.pypirc or setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
+            result = self.config.get(
+                'zest.releaser', 'tag-format')
+            #test the formater
+            result % ({'version':''})
```

Comment:
That is right. Thanks for spotting that.
I was trying to protect against typo like `'%(verion)s'` which I would definitely do.
Only way to  would be to use regex like:
```python
re.search('%\(version\)s', result)
```